test/test_persim_update.py

```diff
@@ -2,7 +2,8 @@
 
 import numpy as np
 from persim import PersistenceImager
-from persim.images import persistence, linear_ramp, uniform, bvncdf, _norm_cdf, _sbvn_cdf, _bvn_cdf
+from persim.images_weights import *
```

Comment:
please no `import *`. I'd prefer seeing `from persim import images_kernels` and then using `images_kernels.gaussian` below to make it clear where everything is coming from.

Reply:
Good point, will update!

> This looks great by the way! 💯 The final thing is adding some bits to the documentation.

What additional docs would you like to see; I updated the notebooks:  `Persistence Images.ipynb` and `Classification with persistence images.ipynb` Do you mean things like index.rst?